app/controllers/admin/topics_controller.rb

```diff
@@ -312,6 +312,43 @@ def assign_team
     end
   end
 
+  def split_topic
+    parent_topic = Topic.find(params[:topic_id])
+    parent_post = Post.find(params[:post_id])
+
+    @topic = Topic.new(
+      name: t('new_discussion_topic_title', original_topic: parent_topic.name, default: "Split from #{parent_topic.name}"),
```

Comment:
Lets include the ID here- so `"Split from #{parent_topic.id}- #{parent_topic.name}"`